cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/help-script.html

```diff
@@ -23,3 +23,39 @@
     but if you specify multiple parameters they must all be named:
 </p>
 <pre>stage name: 'Build', concurrency: 1</pre>
+
+<h3>Environment Variables</h3>
+
+A set of environment variables are made available to all Jenkins Job types, including Workflow Jobs.
+The following is a general list of variables (by name) that are available to all Job types. See the notes
+below the list for Workflow specific details.
+
+<p>
+    <iframe src="${rootURL}/env-vars.html" width="100%"></iframe>
```

Comment:
Ah, [found it](https://github.com/jenkinsci/jenkins/blob/2983905e459a0311a1ca549847663999f4e5cd76/core/src/main/java/hudson/model/Descriptor.java#L831)!